src/index.js

```diff
@@ -1,92 +1,42 @@
-import { driver } from './driver'
-import { schema } from './schema'
 import { ApolloServer } from 'apollo-server-express'
 import express from 'express'
 import cors from 'cors'
 import bodyParser from 'body-parser'
-import GetRequest from './REST/GetRequest'
-import { debug as Debug } from 'debug'
-import validator from 'validator'
-export const debug = Debug('ce-api-debug')
-export const info = Debug('ce-api-info')
-export const warning = Debug('ce-api-warning')
```

Comment:
These logger methods are called from SchemaHelper.js, RequestControlActionCommand.js and mutation.js and don't seem to have been updated to the new location in utils/logger

Reply:
Fixed this issue in `SchemaHelper.js` and `RequestControlActionCommand.js` files.